### PR TITLE
Add tested Istio version for clarity

### PR DESCRIPTION
**Pull request by markusthoemmes**
## Proposed Changes 

We've been asked for "which version of Istio is actually supported" multiple times so I feel like we should clarify this in our docs and update accordingly once new versions stabilize for us.

### CODE REVIEW

**Comment by nak3** [comment]
/assign @ZhiminXiang @JRBANCEL 

I think @ZhiminXiang and @JRBANCEL are familiar with recent tested Istio version.

File: docs/install/installing-istio.md [hunk 1]
```diff
@@ -20,6 +20,12 @@ You need:
 - A Kubernetes cluster created.
 - [`istioctl`](https://istio.io/docs/setup/install/istioctl/) (v1.5.4 or later) installed.
 
+## Supported Istio versions
+
+The current known-to-be-stable version of Istio tested in conjunction with Knative is **v1.5.7**.
+Versions in the 1.5 line above **v1.5.4** should generally be fine too.
+Versions above the 1.5 line are under test but have not stabilized yet.
```
Comment (mattmoor) [inline docs/install/installing-istio.md:27]: It's actually 1.4.x, IIUC net-istio bumped stable before fixing the way serving pinned things.

cc @tcnghia @JRBANCEL

Comment (ZhiminXiang) [reply docs/install/installing-istio.md:27]: yea. currently this is 1.4.10. The plan is to bump to Istio 1.7 soon.

Comment (markusthoemmes) [reply docs/install/installing-istio.md]: Adjusted.

**Pull request merged**